smart_simulation/Notebooks/modeling.ipynb

```diff
@@ -0,0 +1,942 @@
+{
```

Comment:
<p>Put a note of what you'd expect the acf to look like if it was a stationary time series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbpauly/smart-simulation/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Will do</p>